cmd/api/src/database/migration/migrations/v6.1.0.sql

```diff
@@ -10,4 +10,16 @@ CREATE TABLE IF NOT EXISTS oidc_providers
   created_at TIMESTAMP WITH TIME ZONE DEFAULT now(),
 
   UNIQUE (name)
-)
+);
+
+
+-- Add Scheduled Analysis Configs
+INSERT INTO parameters (key, name, description, value, created_at, updated_at)
+  VALUES ('analysis.scheduled',
+        'Scheduled Analysis',
+        'This configuration parameter allows setting a schedule for analysis. When enabled, analysis will only run when the scheduled time arrives',
+        '{"enabled": false, "rrule": ""}',
+        current_timestamp,current_timestamp) ON CONFLICT DO NOTHING;
+
```

Comment:
Nit, we've gotten into a bit of a habit of having comments per migration, didn't know if you wanted to tack something on here for the purpose of the datapipe-status change 🤷